src/item_action.cpp

```diff
@@ -120,9 +129,39 @@ item_action_map map_actions_to_items( player &p )
     return candidates;
 }
 
+std::string item_action_generator::get_action_name( const item_action_id &id ) const
```

Comment:
The last change was from const ref to pure string, because BevapDin said it would be less confusing this way. At least that's how I understood it.